src/Npgsql/NpgsqlTracingOptions.cs

```diff
@@ -1,9 +1,50 @@
+using System;
+using System.Diagnostics;
+
 namespace Npgsql;
 
 /// <summary>
 /// Options to configure Npgsql's support for OpenTelemetry tracing.
-/// Currently no options are available.
 /// </summary>
 public class NpgsqlTracingOptions
 {
-}
\ No newline at end of file
+    internal static NpgsqlTracingOptions? Current { get; set; }
+
+    /// <summary>
+    /// Gets or sets a filter function that determines whether or not to
+    /// collect telemetry on a per <see cref="NpgsqlCommand"/> basis.
+    /// </summary>
+    public Func<NpgsqlCommand, bool>? FilterNpgsqlCommand { get; set; }
```

Comment:
One general thought about this type: since it now mainly has Funcs and Actions, it may be better for it to simply be an abstract base class which the user extends, implementing the methods they want to implement... What do you think?

Reply:
Seems a bit of overkill (if I only want to specify one option, why I have to write a whole class and implementation for every single property) + the way it's now it aligns with the way other providers implement options.

Reply:
OK